MADE.App.Design/Color/Color.cs

```diff
@@ -37,6 +39,33 @@ internal Color(byte a, byte r, byte g, byte b)
             this.B = b;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Color"/> class.
+        /// </summary>
+        /// <param name="hexValue">
+        /// The ARGB or RGB hex value represented as a string.
+        /// </param>
+        public Color(string hexValue)
```

Comment:
Nice improvement.

Reply:
Thanks, I thought I would add this straight into the color class. There is an extension that we could possibly apply to the platform specifics if folks aren't using this class in particular.